### ISSUE TITLE
Natural Forest Plan Form Format

### ISSUE THREAD

**Issue by LeScruf** [mstenta/farm_nfa#140]
There are some additions required to the NF Plan field format - see https://forests.nfa.go.ug/asset/517 as a working example:

1.  Boundaries:
a. Add 'Boundary demarcation' section to include:
   - Year
   - Description of work done
   - File upload of map of work done
b. Date of boundary plan:
    - Change date to month/year format

2. Predominant Vegetation:
- Remove drop down on 'predominant vegetation'
- Add file upload for vegetation map
- Change section title to 'Vegetation'

4. Zonation:
 - Add 'Zonation' section in 'Current purpose' section to include:
   - File upload for zonation map
   - Long (rich) text field for zonation description
   - Date of zonation map

5. Historical context
 - Add 'Historical context' section in 'Current Purpose' as a long (rich) text field

6. Change 'Current Purpose' section name to 'General description'

7. Threats assessment - add section in Mangement issues section as long (rich text).

8. Remove 'ID tags' section (i.e. hide it)

**Comment by pcambra** [comment]
@LeScruf I'm looking at this and I fail to see how this is related to the "Natural Forest Plan Form", it doesn't seem that the fields described are there, the asset linked is a CFR. I don't know what's "NF Plan field format" either, sorry.

**Comment by LeScruf** [comment]
Yes - it's a linked CFR - but part of a natural forest plan - apologies if this I described this in a confusing manner.

**Comment by pcambra** [comment]
@LeScruf does this mean that the changes requested do not apply for Plantation Forests?

**Comment by pcambra** [comment]
@LeScruf this has been now deployed and is ready for you to take a look :)
